src/content/index.js

```diff
@@ -28,13 +29,9 @@ function removeDuplicatePlaces(placesAlreadyScraped, newPlaces) {
 
 function updateStorageWithNewPlaces(newPlaces) {
     chrome.storage.local.get(['lastPlacesScraped'], ({ lastPlacesScraped }) => {
-        const filteredPlaces = removeDuplicatePlaces(lastPlacesScraped, newPlaces);
-        let updatedPlaces = filteredPlaces;
-        if (Array.isArray(lastPlacesScraped)) {
-            updatedPlaces = filteredPlaces.concat(lastPlacesScraped);
-        }
+        const existing = Array.isArray(lastPlacesScraped) ? lastPlacesScraped : [];
         chrome.storage.local.set({
-            lastPlacesScraped: updatedPlaces,
+            lastPlacesScraped: (removeDuplicatesWith(existing, newPlaces, 'name')).concat(existing),
```

Comment:
do you need those extra parens? around ```(removeDuplicatesWith(existing, newPlaces, 'name'))``` ?

Reply:
It makes it more readable for me, but I can remove it if it's not according to our eslint standards

Reply:
ok whatever you prefer. personal preference